types/enzyme/index.d.ts

```diff
@@ -430,7 +430,15 @@ export interface ShallowWrapper<P = {}, S = {}> extends CommonWrapper<P, S> {
      * NOTE: can only be called on wrapper of a single non-DOM component element node.
      * @param [options]
      */
-    dive<P2, S2>(options?: ShallowRendererProps): ShallowWrapper<P2, S2>;
+    dive<P2, S2>(options?: ShallowRendererProps): ShallowWrapper<any, any>;
```

Comment:
Don't believe this line needs to be changed

Reply:
indeed it does not need to